ods_ci/run_interop.sh

```diff
@@ -0,0 +1,424 @@
+#!/bin/bash
```

Comment:
why this file is not under the folder `ods_ci/build`?

Reply:
it is just an option to use the image outside of build folder.